linuxNew/README.md

```diff
@@ -114,11 +123,13 @@ rpm-build-4.16.1.3-1.fc33.x86_64
 
 Consider this RPM build where x86_64 is the build hosts' architecture.
 Download the release blobs and associated sources.
+Suppose build rpm for jdk11 for target architecture `x86_64`
 
 ```shell
-spec=$(pwd)/temurin-11-jdk.spec
+cd linuxNew/jdk/redhat/src/main/packaging/temurin/11
```

Comment:
Won't be linuxNew much longer I suppose

Reply:
yeah, PR of renaming "linuxNew" to "linux" need rebase this PR first and updates these references.